main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        Genre(name="Western"),
+        Genre(name="Action"),
+        Genre(name="Dramma")
+    ]
+    Genre.objects.bulk_create(genres)
+
+    actors = [
+        Actor(first_name="George", last_name="Klooney"),
+        Actor(first_name="Kianu", last_name="Reaves"),
+        Actor(first_name="Scarlett", last_name="Keegan"),
+        Actor(first_name="Will", last_name="Smith"),
+        Actor(first_name="Jaden", last_name="Smith"),
+        Actor(first_name="Scarlett", last_name="Johansson")
+    ]
+
+    Actor.objects.bulk_create(actors)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+
+    actors = Actor.objects.all()
+
+    for actor in actors:
+        if actor.first_name == "Scarlett":
+            actor.delete()
```

Comment:
Use ORM for this operation, do not loop through every object in QuerySet, it is not efficient.